src/main/java/com/intershop/oms/test/servicehandler/omsdb/v1/OMSDbHandlerV1.java

```diff
@@ -2762,8 +2801,16 @@ public String getDispatchError(Long dispatchRef)
     @Override
     public boolean getPlatformConfigShowgMaxReturnQty()
     {
-        return runDBStmtBoolean("SELECT \"showMaxReturnQuantity\" FROM \"PlatformConfigDO\"", "showMaxReturnQuantity",
-                        true);
+        return runDBStmtBoolean("SELECT \"showMaxReturnQuantity\" FROM \"PlatformConfigDO\"", "showMaxReturnQuantity", true);
+    }
+
+    @Override
+    public List<Long> getAllReturnIdsForOrder(OMSOrder order)
+    {
+        String query = "SELECT id FROM oms.\"ReturnDO\" WHERE \"orderRef\" in (?)";
```

Comment:
maybe add "ORDER BY id" to ensure a stable list?

Reply:
done